mne/evoked.py

```diff
@@ -410,21 +410,21 @@ def plot_image(self, picks=None, exclude='bads', unit=True, show=True,
             sphere=sphere)
 
     @copy_function_doc_to_method_doc(plot_evoked_topo)
-    def plot_topo(self, layout=None, layout_scale=0.945, color=None,
-                  border='none', ylim=None, scalings=None, title=None,
-                  proj=False, vline=[0.0], fig_background=None,
-                  merge_grads=False, legend=True, axes=None,
+    def plot_topo(self, layout=None, layout_scale=0.945,
+                  color=None, border='none', ylim=None, scalings=None,
+                  title=None, proj=False, vline=[0.0], fig_background=None,
+                  merge_grads=False, legend=True, axes=None, exclude=None,
```

Comment:
you should add it at the end before `show` so there is minimal chance of breaking existing code

Reply:
Alright, Added that in commit https://github.com/mne-tools/mne-python/pull/9278/commits/3630b0e68bfe91801d375984dc5b4690c2edf2d4